internal/oci/container.go

```diff
@@ -350,14 +391,105 @@ func (c *Container) exitFilePath() string {
 // IsAlive is a function that checks if a container's init pid exists.
 // It is used to check a container state when we don't want a `$runtime state` call
 func (c *Container) IsAlive() bool {
-	if _, err := findprocess.FindProcess(c.state.Pid); err != nil {
-		logrus.Errorf("container %s not running: %v", c.id, err)
+	_, err := c.pid()
+	if err != nil {
+		logrus.Errorf("checking if pid of %s is running failed: %v", c.id, err)
 		return false
 	}
 
 	return true
 }
 
+// Pid is the external, locking function that returns the container's
```

Comment:
Since the name starts with an uppercased letter, it's obvious that it is an external function, so you can omit saying it.
You can probably omit the "locking" part, too, since it's an implementation detail.